script/fork-helpers/SimulateVote.s.sol

```diff
@@ -109,6 +109,22 @@ contract SimulateVote is Script, DeploymentFixtures, ForkHelpersCommon {
         IBurner burner = IBurner(locator.burner());
 
         vm.startBroadcast(admin);
+
+        accounting.revokeRole(accounting.SET_BOND_CURVE_ROLE(), address(csm));
+        accounting.revokeRole(
+            accounting.SET_BOND_CURVE_ROLE(),
+            accounting.getRoleMember(accounting.SET_BOND_CURVE_ROLE(), 0)
+        );
+        csm.grantRole(
+            csm.CREATE_NODE_OPERATOR_ROLE(),
+            upgradeConfig.permissionlessGate
+        );
+        csm.grantRole(
+            csm.CREATE_NODE_OPERATOR_ROLE(),
+            upgradeConfig.vettedGate
+        );
+        csm.grantRole(csm.SET_BOND_CURVE_ROLE(), upgradeConfig.vettedGate);
```

Comment:
We need to grant this role to the CSM committee as well

Reply:
in the deploy script as well
thanks, missed this